<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrendChartSetup</class>
 <widget class="QWidget" name="TrendChartSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#9fa8da;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,5,0,0">
      <item>
       <widget class="QLabel" name="trend_setup_l">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>912</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Trend Setup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="data_1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox::enabled{background-color:white;}
QCheckBox::disabled{background-color:#e5e5e5;}
QCheckBox::unchecked{background-color:#9fa8da}QCheckBox::checked{background-color:#9fa8da}
QCheckBox::indicator {	width : 30px; height : 30px;}</string>
          </property>
          <property name="text">
           <string>Data 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="data_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox::enabled{background-color:white;}
QCheckBox::disabled{background-color:#e5e5e5;}
QCheckBox::unchecked{background-color:#9fa8da}QCheckBox::checked{background-color:#9fa8da}
QCheckBox::indicator {	width : 30px; height : 30px;}</string>
          </property>
          <property name="text">
           <string>Data 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="data_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox::enabled{background-color:white;}
QCheckBox::disabled{background-color:#e5e5e5;}
QCheckBox::unchecked{background-color:#9fa8da}QCheckBox::checked{background-color:#9fa8da}
QCheckBox::indicator {	width : 30px; height : 30px;}</string>
          </property>
          <property name="text">
           <string>Data 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="data_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox::enabled{background-color:white;}
QCheckBox::disabled{background-color:#e5e5e5;}
QCheckBox::unchecked{background-color:#9fa8da}QCheckBox::checked{background-color:#9fa8da}
QCheckBox::indicator {	width : 30px; height : 30px;}</string>
          </property>
          <property name="text">
           <string>Data 4</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="trend_setup_ok_pb">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#0d47a1;
color:white;</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
